### ISSUE TITLE
[Feature]: Allow Jinja-based variable in the inventory plugin?

### ISSUE THREAD

**Issue by raisedadead** [linode/ansible_linode#372]
### Description

Here is a sample inventory config with some Jinja-based templating:

```yaml
plugin: linode.cloud.instance
api_token: "{{ lookup('env', 'LINODE_API_TOKEN_BUT_AS_A_DIFFERENT_NAME') }}"

keyed_groups:
  - key: tags
    separator: ''

groups:
  manager: "'leader' in (tags|list)"
  worker: "'worker' in (tags|list)"

compose:
  ansible_ssh_host: ipv4[0]

```

I noticed that it doesn't work, but the below does:

```yaml
plugin: linode.cloud.instance
# omit the api_token key, but set the LINODE_API_TOKEN as a OS Environment Variable

keyed_groups:
...
```

OR this works too:

```yaml
plugin: linode.cloud.instance
api_token: xxxxxxxxxxxxxxxxxxxxxxxxxxxx

keyed_groups:
...
```

Do you think this is intentional? I can see that the `community.general.linode` does support the Jinja-based syntax.

**Comment by lgarber-akamai** [comment]
@raisedadead Sorry for the delay on this; this feature has been merged and was released alongside [`v0.18.3`](https://github.com/linode/ansible_linode/releases/tag/v0.18.3).

**Comment by raisedadead** [comment]
Thanks, @lgarber-akamai - I appreciate it.
